package-lock.json

```diff
@@ -157,6 +157,43 @@
       "integrity": "sha512-AbK4ZPP0MMXMuUzfxQb3f71kA++kV9wao0ps5jOsvaHXk1SjS+0Cxsfae+wqfwyvy1oyF9N6Kl8zYqtJizQmjA==",
       "dev": true
     },
+    "@firebase/app": {
```

Comment:
I don't see a reason this file should have been updated... can this be reverted?